build_tools/github_actions/ci_build_cmake.sh

```diff
@@ -27,21 +27,25 @@ fi
 
 LLVM_BUILD_DIR="$1"
 STABLEHLO_BUILD_DIR="$2"
+CMAKE_BUILD_TYPE="${CMAKE_BUILD_TYPE:-RelWithDebInfo}"
+
+# Turn on building Python bindings
+STABLEHLO_ENABLE_BINDINGS_PYTHON="${STABLEHLO_ENABLE_BINDINGS_PYTHON:-OFF}"
 
 # Configure StableHLO
 cmake -GNinja \
   -B"$STABLEHLO_BUILD_DIR" \
   -DLLVM_ENABLE_LLD=ON \
-  -DCMAKE_BUILD_TYPE=Release \
+  -DCMAKE_BUILD_TYPE="${CMAKE_BUILD_TYPE}" \
   -DLLVM_ENABLE_ASSERTIONS=ON \
   -DMLIR_DIR="$LLVM_BUILD_DIR/lib/cmake/mlir" \
   -DCMAKE_CXX_COMPILER=clang++ \
   -DCMAKE_CXX_COMPILER_LAUNCHER=ccache \
   -DCMAKE_C_COMPILER=clang \
   -DCMAKE_C_COMPILER_LAUNCHER=ccache \
   -DSTABLEHLO_ENABLE_STRICT_BUILD=ON \
-  -DSTABLEHLO_ENABLE_BINDINGS_PYTHON=ON
+  -DSTABLEHLO_ENABLE_BINDINGS_PYTHON="${STABLEHLO_ENABLE_BINDINGS_PYTHON}"
 
 # Build and Test StableHLO
-cd "$STABLEHLO_BUILD_DIR"
+cd "$STABLEHLO_BUILD_DIR" || exit
```

Comment:
This was an error ShellCheck (https://www.shellcheck.net/) gave me in my editor.
Could be yet another lint to apply :)

Reply:
Neat! I personally like setting `set -o errexit` (or `set -e` for short) at the very top of the script. In this case if `cmake` fails we probably don't want to call `ninja` either.

While we're on this, I also prefer `cmake --build "$STABLEHLO_BUILD_DIR" --target check-stablehlo-ci`, but calling `cd`ing and calling `ninja` is also fine.

Reply:
Yes we should probably apply best practices for shellcheck;
I also like the `cmake --build` approach but it's more "new"